andela/activo-api
# #158625122 refactor get resource model operation

#### What does this PR do?
- Refactor get model operation implementation to encourage DRY principle
#### Description of Task to be completed?
- Abstract checking if a resource exists into the model operation so as to avoid writing code to raise validation error anytime a resource is fetched with the get method
#### How should this be manually tested?
- Create a local copy with the following commands: `git fetch origin ch-refactor-check-resource-exists-158625122:ch-refactor-check-resource-exists-158625122`, then `git checkout ch-refactor-check-resource-exists-158625122`
- Run the tests with the command `pytest` to confirm that there was no regression introduced
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#158625122](https://www.pivotaltracker.com/story/show/158625122)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
